department-of-veterans-affairs/content-build
# [VA-14940] Add front End template skeleton for Facility content type for VBA

## Description

This pull request adds the template skeleton for the VBA Facility pages. "Template Skeleton" refers to getting as many elements from the query that are currently available (in this case, everything but the centralized content in accordions), and bringing them to their needed page elements.

relates to [#14940](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14940)

## Testing done & Screenshots

Visual, see below.

<img width="871" alt="Screen Shot 2023-10-31 at 5 44 34 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/10790736/2895757a-0693-4836-8dbd-f30f6374d44b">
<img width="867" alt="Screen Shot 2023-11-01 at 7 29 49 PM" src="https://github.com/department-of-veterans-affairs/content-build/assets/10790736/ffda12f7-c89d-4f73-a949-5be758fcff87">


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

1. Set up and confirm local build works
   - [ ] Run `yarn build --pull-drupal --drupal-address=https://cms-txcrk8bym4a0ivw3xfr8xvdhezmls36f.demo.cms.va.gov/ username=max.antonucci@agile6.com password=drupal8` to build site with needed data. Confirm the build is successful.
   - [ ] Run `yarn serve` to view site locally.
   - [ ] Confirm VBA Facility page is active by visiting `http://localhost:3002/vba-facilities/locations/seattle-regional-office/`
   - [ ] Confirm that the existing page elements are styled properly and in the proper page layout.
2. Confirm that the correct data from the query is being used in the skeleton
   - [ ] Check that the correct information is currently being pulled from the query and used correctly in the template
   - [ ] Check if any additional data could be pulled from the query and used in the template

## Acceptance criteria

- [ ] All QA steps pass and are successfully checked off.
- [ ] Laura has completed an accessibility review.

## Definition of done
